docs/changelog.md

```diff
@@ -1,6 +1,7 @@
 # Next release
 
 * Drop Python 3.5 support
+* Preserves file attributes during the `copy` statements in `repos`
```

Comment:
Please add a reference to the issue : "Preserves file attributes [...]. Fix #217"